src/index.ts

```diff
@@ -85,26 +85,21 @@ type Style = {
 
 const styles = new WeakMap<Element, Style>()
 
-class MarkdownButtonElement extends HTMLElement {
-  constructor() {
-    super()
-    const apply = () => {
-      const style = styles.get(this)
-      if (!style) return
-      applyStyle(this, style)
-    }
-    this.addEventListener('keydown', keydown(apply))
-    this.addEventListener('click', apply)
-  }
-
+export class MarkdownButtonElement extends HTMLElement {
   connectedCallback() {
     if (!this.hasAttribute('role')) {
       this.setAttribute('role', 'button')
     }
+    this.addEventListener('keydown', keydown(this.click))
```

Comment:
What's the argument for moving these two `addEventListener` calls out of the `constructor`? I know that may not be our typical style but for the sake of minimizing changes I feel like we should leave them where they are.

Reply:
My thought was that since this is the convention in our codebase it would be less surprising for subclass implementers. Can definitely see your point though, I'll change it back.